LICENSE

```diff
@@ -1,11 +1,11 @@
 MIT License
 
-Copyright (c) 2019 Authors of Tshitoyan et al. Nature (2019)
+Copyright(c) 2019 Authors of Tshitoyan et al. Nature(2019)
 
 Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
+of this software and associated documentation files(the "Software"), to deal
 in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
```

Comment:
I would refrain from changing their license file, or generally to making any unnecessary changes to their code that we don't intend to upstream.

Reply:
Sorry, autopep8 was applied to this file by mistake.